deploy/kubernetes/setup-csi-snapshotter.yaml

```diff
@@ -72,7 +72,7 @@ spec:
       serviceAccount: csi-snapshotter
       containers:
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.0.1
+          image: quay.io/k8scsi/csi-provisioner:v1.1.0
```

Comment:
should the examples here just use canary so we don't have to remember to keep updating it every release?

Reply:
Canary images may not work with each other before they are released. For example, I may have modified external-snapshotter APIs in the master but have not got time to update external-provisioner code yet, so the canary images won't work with each other.